ansible-scylla-node/defaults/main.yml

```diff
@@ -175,6 +175,12 @@ gpfs_prefer_local: "true"
 # probe for IO figures on one node and reuse the figures on all nodes (recommended: True)
 scylla_io_probe: True
 
+# If set to True, scylla_io_probe must be True and dc variable must be defined in the inventory
+# for each one of the nodes.
+# This will make sure that scylla_io_setup will be executed in one node from each datacenter
+# and then the configuration files will be propagated for the other nodes of each datacenter.
+scylla_io_probe_dc_aware: False
```

Comment:
Why false by default?

Reply:
My concern is that for any already existing cluster which don't use `gossipingPropertyFileSnitch`, if this new variable is set to `True` by default, as soon as we run an update task in any of these clusters, this will update the `io_properties` for this cluster and in this case the `DC` variable might not be present, what would cause the playbook execution to fail, or the `DC` variable might have wrong information, and in this case the playbook wouldn't fail but we would end up with wrong io settings.

Reply:
Patch description added